hack/dataplaneParser/iptable/iptable.go

```diff
@@ -0,0 +1,114 @@
+package iptable
+
+import (
+	"os"
+	"time"
+
+	"github.com/Azure/azure-container-networking/npm/util"
+	"golang.org/x/sys/unix"
+	"k8s.io/apimachinery/pkg/util/wait"
+)
+
+type Iptables struct {
+	Name   string
+	Chains map[string]*IptablesChain
+}
+
+type IptablesChain struct {
+	Name  string
+	Data  []byte
+	Rules []*IptablesRule
+}
+
+type IptablesRule struct {
+	Protocol string
+	Target   *Target
+	Modules  []*Module
+}
+
+type Module struct {
```

Comment:
module and target will not be referred alone from outside, so lets make them start with lower case too, that way we can be sure that they will be always be used through the IptableRule